dpctl/__init__.py

```diff
@@ -1,43 +1,43 @@
+##===--------------- _memory.pyx - dpctl interface ------*- Cython -*------===##
+##
+##                      Data Parallel Control (dpctl)
+##
+## Copyright 2020 Intel Corporation
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##    http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##
+##===----------------------------------------------------------------------===##
+##
+## \file
+## This top-level dpctl module.
+##
+##===----------------------------------------------------------------------===##
 '''
-    Python Data Parallel Processing Library (PyDPPL)
+    Data Parallel Control (dpctl)
 
-    PyDPPL provides a lightweight Python abstraction over DPC++/SYCL and
+    Dpctl provides a lightweight Python abstraction over DPC++/SYCL and
```

Comment:
```suggestion
    dpCtl provides a lightweight Python abstraction over DPC++/SYCL and
```

Reply:
Done. The module is `dpctl` and the project `dpCtl`